app/views/group_assignments/show.html.erb

```diff
@@ -89,7 +88,6 @@
     <% else %>
       <% if @group_assignment_repos.present? %>
         <%= render partial: 'group_assignments/filterable_group_assignment_repo_list', locals: { param_name: :page } %>
-        <%= render partial: 'shared/pagination', locals: { collection: @group_assignment_repos } %>
```

Comment:
Did we mean to remove this? It looks like this removed pagination from this view.

Reply:
Yup, in fact this should never have actually been here, because it's included in the `group_assignments/filterable_group_assignment_repo_list` partial, where it's re-rendered accordingly as the results returned from the search query change